samples/node/routes/weather.js

```diff
@@ -9,19 +9,26 @@ const uuidV4 = require('uuid/v4');
 const sha1 = require('sha1');
 
 exports.requestCards = function(req, res) {
-    if (!req.body.tokens) {
-        res.status(400).send("Missing tokens field");
+    let tokenZip = (req.body.tokens && req.body.tokens.zip) ? req.body.tokens.zip : null;
```

Comment:
There's no reason to use a ternary here.

```
let tokenZip = req.body.tokens && req.body.tokens.zip;
```

tokenZip will be falsy if tokens or tokens.zip is falsy.

Ditto for configZip.

Reply:
I'm actually grabbing the array here, not just resolving to a boolean

Reply:
Forget the ditto for configZip, however.  That still needs to be ternary (or something different), since it's converting the string to an array of the string.

Reply:
Alright, check the new changes.  Updated the JS and fleshed out config+validation more.